index.bs

```diff
@@ -598,6 +598,7 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
   <dt><dfn dict-member for="MediaTrackConstraintSet"><code>pan</code></dfn></dt>
   <dd>A value of true is normalized to a value of empty {{ConstrainDouble}}.
       A value of false is normalized to a value of undefined.
+      An empty {{ConstrainDouble}} implies no constraints but a <a>pan capability</a> request (ideal in basic constraints and required in advanced constraints).
```

Comment:
What does ideal in basic constraints and required in advanced constraints mean?
My understanding is that we don't want to have a way to require the capability.
Since advanced constraints are for required constraints only, pan:{} in advanced should only mean no constraints and nothing else. No capability request. Otherwise we do not solve the fingerprinting issue we want to address by not having required constraints.

To make it clear, allowing pan:{} or pan:true to mean a required capability request in an advanced set allows fingerprinting by adding additional constraints and checking if the other constraints were ignored.
For example: getUserMedia({video: {advanced: [{pan:true, width:10}]}).
In this case, in Chromium (which supports crop-and-scale), getting a track with a width different than 10 means that pan is not supported, which is basically equivalent to having an OverconstrainedError if the capability request was required in the basic set.

IMO, now that we want to treat capability requests similarly to an ideal value, such requests should be only in the basic set, which is the only one that allows ideal values.

Reply:
> To make it clear, allowing pan:{} or pan:true to mean a required capability request in an advanced set allows fingerprinting by adding additional constraints and checking if the other constraints were ignored.

@guidou This is after the user has granted permission, so do we care? Can't they already check whether they got ptz permission? If they were, they know the user has a ptz device. If they weren't, then most likely not¹.

> ... which is basically equivalent to having an OverconstrainedError if the capability request was required in the basic set.

They're not equivalent, because `OverconstrainedError` is thrown before any permission prompt, which can be used by tracking libraries without a user's knowledge or permission.

`advanced` constraints cannot throw `OverconstrainedError` by design, so I don't see a significant fingerprint risk.

---
<sub>1. Sure some user agents may offer users a way to downgrade requests to non-ptz, but I doubt many would.</sub>